ci-operator/step-registry/telcov10n/functional/cnf-network/send-slack-notification/telcov10n-functional-cnf-network-send-slack-notification-commands.sh

```diff
@@ -17,26 +32,47 @@ if [[ ! -f "$WEBHOOK_URL_FILE" ]]; then
   exit 1
 fi
 
+if [[ -z "$CLUSTER_NAME_FILE" ]]; then
```

Comment:
in line 10 you check if there is a file and in this line you check if the var is empty
I think it should be handled a bit batter and maybe much earlier in the script